app/jobs/bulkrax/schedule_relationships_job_decorator.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# OVERRIDE Bulkrax v9.0.2 to alter the query so we kick off the relationship jobs at the correct time
+
+module Bulkrax
+  module ScheduleRelationshipsJobDecorator
+    def perform(importer_id:)
+      importer = Importer.find(importer_id)
+
+      processed_entries =
+        Bulkrax::Status
+        .where(statusable_id: importer.entries.pluck(:id))
+        .where(runnable_id: importer.last_run.id)
+        .where(status_message: ['Complete', 'Failed', 'Skipped'])
```

Comment:
I'm not positive off the top of my head, but I don't think this encompasses all possible statuses. Either way, what we're truly after is "entries that are not pending", so I recommend something like: 

```suggestion
        .where.not(status_message: 'Pending')
```
It may actually be `.where.not(status_message: nil)` but you get the idea

Reply:
Thanks sounds good to me, committing it.